From 264e2d7c27d0f690b67e4cacf7b19dfeb7eeeca0 Mon Sep 17 00:00:00 2001
From: Ricardo Salveti de Araujo <ricardo.salveti@canonical.com>
Date: Fri, 27 Jun 2014 16:36:48 -0300
Subject: [PATCH] daemon-conf.c: changing default rlimit_rttime value to 200 ms

Since rtkit v11, the top limit for rttime is 200 ms (previously it
was wrongly limited to 2 seconds).

Signed-off-by: Ricardo Salveti de Araujo <ricardo.salveti@canonical.com>
---
 src/daemon/daemon-conf.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: pulseaudio/src/daemon/daemon-conf.c
===================================================================
--- pulseaudio.orig/src/daemon/daemon-conf.c
+++ pulseaudio/src/daemon/daemon-conf.c
@@ -139,7 +139,7 @@ static const pa_daemon_conf default_conf
    ,.rlimit_rtprio = { .value = 9, .is_set = true }    /* One below JACK's default for the server */
 #endif
 #ifdef RLIMIT_RTTIME
-   ,.rlimit_rttime = { .value = PA_USEC_PER_SEC, .is_set = true }
+   ,.rlimit_rttime = { .value = 200*PA_USEC_PER_MSEC, .is_set = true } /* rtkit's limit is 200 ms */
 #endif
 #endif
 };
